README.md

```diff
@@ -153,35 +162,72 @@ ProfileFolio simplifies professional profile management by generating customized
 }
 ```
 
-2. Add this sample data in collection `skills`
+**2. Add this sample data in collection `skills`**
+___
 
 ```json
 [
   {
     "_id": {
       "$oid": "667b888850aa1b4215433ff8"
     },
-    "name": "typescript"
+    "name": "typescript",
+    "level": "intermediate",
+    "keywords": [""]
   },
```

Comment:
Why is keywords addded?

Reply:
Skills db would hold a collection of skills, I followed this and added the keywords section.
```bash
type Skill struct {
	Name     string               `json:"name"`
	Level    string               `json:"level"`
	Keywords []primitive.ObjectID `json:"keywords,omitempty"`
}
```

Reply:
Okay cool, but omit empty already is doing it's job ... You can either add something meaningful or leave it. This empty doesn't make sense in this scenario, because it is my default

Reply:
Ah I see I see, I think I will just leave it blank then for now